docs/src/quickstart.md

```diff
@@ -0,0 +1,115 @@
+# Quickstart
+
+## Terminology
+CUDA being the most popular GPU programming environment, we take it as a
+reference for defining terminology in KA. A *workgroup is called a block in
```

Comment:
```suggestion
Because CUDA is the most popular GPU programming environment, we can use it as a
reference for defining terminology in KA. A *workgroup* is called a block in
```

This sounded a little clunky. It was also missing an `*`